experiment/measurer/measure_manager.py

```diff
@@ -678,32 +678,32 @@ def initialize_logs():
         'subcomponent': 'measurer',
     })
 
-def consume_snapshots_from_response_queue(response_queue, queued_snapshots
-                                          ) -> List[models.Snapshot]:
+
+def consume_snapshots_from_response_queue(
+        response_queue, queued_snapshots) -> List[models.Snapshot]:
     """Consume response_queue, allows reeschedule objects to reescheduled, and
     return all measured snapshots in a list."""
     measured_snapshots = []
     while True:
         try:
             response_object = response_queue.get_nowait()
-            match type(response_object):
-                case datatypes.ReescheduleRequest:
-                    # Need to reeschedule measurement task, remove from the set
-                    snapshot_identifier = (response_object.trial_id,
-                                           response_object.cycle)
-                    queued_snapshots.remove(snapshot_identifier)
-                    logger.info(
-                        'Reescheduling task for trial %s and cycle %s',
-                        response_object.trial_id, response_object.cycle)
-                case models.Snapshot:
-                    measured_snapshots.append( response_object )
-                case _:
-                    logger.error('Type of response object not mapped! %s',
-                                 type(response_object))
+            if isinstance(response_object, datatypes.ReescheduleRequest):
```

Comment:
I think it makes more sense to have one data type, with a field to request reschedule.

Reply:
This might be a little tricky to do, because before having reeschedules, we used to return a models.Snapshot type in some of our functions. We might not want to add a reeschedule field to it, right?

Reply:
No, but we can return the response object instead can't we?